libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/hwdef.dat

```diff
@@ -0,0 +1,187 @@
+# hw definition file for processing by chibios_pins.py
+# for BrainFPV RADIX 2 HD
+
+# MCU class and specific type
+MCU STM32H7xx STM32H750xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1117
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 16000000
+
+MCU_CLOCKRATE_MHZ 480
+
+env OPTIMIZE -O2
+
+define HAL_WITH_EKF_DOUBLE 0
+
+STM32_ST_USE_TIMER 13
+define CH_CFG_ST_RESOLUTION 16
+
+# internal flash is off limits
+FLASH_SIZE_KB 128
+FLASH_RESERVE_START_KB 0
+define HAL_FLASH_PROTECTION 1
+
+EXT_FLASH_SIZE_MB 16
+EXT_FLASH_RESERVE_START_KB 4096
+EXT_FLASH_RESERVE_END_KB 0
+
+# Supports upto 2 MBits/s
+CANFD_SUPPORTED 2
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6
+
+# I2C1 for baro
+PB6 I2C1_SCL I2C1 PULLUP
+PB7 I2C1_SDA I2C1 PULLUP
+
+# Analog inputs
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PA6 BATT_CURRENT_SENS ADC1 SCALE(1)
+PC1 RSSI_ADC ADC1
+
+define HAL_BATT_MONITOR_DEFAULT 4
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 3
+define HAL_BATT_VOLT_SCALE 17.6
+define HAL_BATT_CURR_SCALE 28.5
+define BOARD_RSSI_ANA_PIN 11
+
+# Debug
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART1 (HD OSD)
+PB15 USART1_RX USART1
+PB14 USART1_TX USART1
+
+# USART2 (GPS)
+PD6 USART2_RX USART2
+PD5 USART2_TX USART2
+
+# USART3 (RC input)
+PB11 USART3_RX USART3
+PD8 USART3_TX USART3
+
+# UART4 (ESC telemetry)
+PB8 UART4_RX UART4 NODMA
+PA0 UART4_TX UART4 NODMA
```

Comment:
Can define this as ESC telemetry protocol

Reply:
I didn't set it to default to ESC telemetry. Some users may want it and the RX4 signal is available on the 4-in-1 ESC connector, but it isn't typically used in my experience.